content/ctl_chef_solo.md

```diff
@@ -190,6 +190,19 @@ chef ALL=(ALL) NOPASSWD: /usr/bin/chef-solo
 where `chef` is the name of the non-root user. This would allow
 chef-solo to run any command on the node without requiring a password.
 
+{{< warning >}}
+
+The configuration for the 'chef' user as described above will still 
+going to have root-level access by calling system commands from a Chef 
+script file with sudo privileges. 
+
+To prevent such kind of security issues, it is highly recommended to create 
+an alternate approach for the cases where 'chef' user should not have 
+built-in root level access.
+
+{{< warning >}}
```

Comment:
```suggestion
{{< /warning >}}
```

This needs to be a closing tag, otherwise you're opening a second warning tag within the first. 

![Screen Shot 2020-09-23 at 10 57 09 AM](https://user-images.githubusercontent.com/22962/94050750-8ca31180-fd8b-11ea-86d2-3a7cbd4b2fb7.png)